.github/workflows/build-git-annex-debianstandalone.yaml

```diff
@@ -124,6 +124,15 @@ jobs:
         run: |
           sudo dpkg -i git-annex*.deb
 
+      - name: Set up SSH target
+        shell: bash
+        # TODO: Drop the release condition once 0.13.1 is released.
+        run: |
+          if [ "${{ matrix.version }}" != "release" ]; then
+            bash tools/ci/prep-travis-forssh.sh;
```

Comment:
Doh, this has no chance of working given this isn't getting executed from DataLad's source directory.

Reply:
Just clone or use https://github.com/actions/checkout 
```
- uses: actions/checkout@v2
  with:
    # Repository name with owner. For example, actions/checkout
    # Default: ${{ github.repository }}
    repository: ''

    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''
...
```
the main repo to get access to the script(s)?